packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/streamProvider.test.ts

```diff
@@ -20,20 +19,26 @@ Object.defineProperty(window, 'TextDecoder', {
   value: TextDecoder,
 });
 
-const mockGet = jest.fn().mockImplementation(() => {
+jest.mock('aws-amplify/auth', () => {
+  const originalModule = jest.requireActual('aws-amplify/auth');
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Is this still needed?

Reply:
Will need to pass a generic, or could replace with spying